Install all the things, slap some styles onto that App

marwinburesch opened the pull request.

marwinburesch committed:
Remove starter code

marwinburesch committed:
Install react-router and styled-components

marwinburesch committed:
Add prettierrc

marwinburesch committed:
Import font

marwinburesch committed:
Add GlobalStyles and heading

marwinburesch committed:
Add basic components

macfrei commented on src/App.js:
```diff
@@ -1,25 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+import styled from 'styled-components';
```
Could we import from `styled-components/macro` please? :)

macfrei commented on src/GlobalStyles.js:
```diff
@@ -0,0 +1,19 @@
+import { createGlobalStyle } from 'styled-components';
```
Here as well

macfrei commented on src/Header.js:
```diff
@@ -0,0 +1,50 @@
+export default function Header() {
+    return (
+        <>
+            <Container class="page-header round-top shadow">
```
Do we need those class names?

macfrei commented on src/Header.js:
```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
```
Here as well :)

macfrei commented on src/Header.js:
```diff
@@ -0,0 +1,50 @@
+
+export default function Header() {
+    return (
+        <>
```
I don't think we'll need those fragments here as the elements are within the `Container`

macfrei commented on src/Page.js:
```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
```
Here as well :D

macfrei reviewed (changes requested):
I added some comments, mostly small things. It'd be great if you could import from `styled-components/macro` instead of `styled-components` :)

marwinburesch committed:
Fix StyledComponents imports

marwinburesch committed:
Fix class prop, redundant fragments

marwinburesch commented:
> I added some comments, mostly small things. It'd be great if you could import from `styled-components/macro` instead of `styled-components` :)

Fixed
Thank you very much!

marwinburesch commented on src/Header.js:
```diff
@@ -0,0 +1,48 @@
+        <Container>
+            <h1>onemoreminiround</h1>
+            <Button class="menu-button">
+                <svg
+                    class="icon"
+                    xmlns="http://www.w3.org/2000/svg"
+                    fill="none"
+                    viewBox="0 0 24 24"
+                    stroke="currentColor"
+                >
+                    <path
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
+                        stroke-width="2"
+                        d="M4 6h16M4 12h16M4 18h16"
+                    />
+                </svg>
```
I am considering to just import this, also I just saw a `class` 😱

macfrei replied:
:O :O :O

marwinburesch replied:
Imma fix this later
